### DISCUSSION TITLE
Output Size Limit and Export Disparity

### DISCUSSION THREAD

**Discussion by Ryudoadema** [nagadomi/nunif#257] [Q&A]
@nagadomi If I am using "Export Disparity" using a 4k source, and I select say 360x640 for "Output Size Limit"- will the depthmap use the 4k source for mapping or the 360x640 output? I just want the smallest file size for the rgb output since I don't use the rgb folder, while getting the highest quality depthmap images using a 4k source.

**Comment by nagadomi** [comment]
No. Output Size Limit is the limit of the final output image (SBS, anaglyph etc).

**Reply by nagadomi** [reply]
You can resize the video beforehand or use `-vf scale` option https://github.com/nagadomi/nunif/discussions/227.
Note that if the input resolution is too small, it will also affect the depth.

**Reply by Ryudoadema** [reply]
Well it does have the effect of lowering the rgb image size/resolution as well during export disparity. I will have to assume it does this AFTER the depthmap image is processed?

**Reply by nagadomi** [reply]
No, -vf scale is input resizing. It is applied before the depth estimation.
However, the input image for depth estimation is pre-resized to Depth Resolution (e.g. height=518px), so if it is larger than that,  it should not be practically affected.

**Reply by Ryudoadema** [reply]
Sorry, not sure what if I am understanding. I will have to do a test using output size limit very low, and compare the depth image to that without using output size limit. I'll come back to share the result when I can get to it.
